docs/developer-docs/docs/build-a-keychain/operate-a-keychain/create-a-keychain.md

```
@@ -107,46 +113,81 @@ A Keychain Writer is an account that can write Keychain results (public keys and
+   ```bash
+   wardend query bank balances my-keychain-writer-name
+   ```
 
    :::tip
-   Only the Keychain Writer address will be able to publish signatures and public keys on behalf of the Keychain.
+   In this example, we used `$(wardend keys show -a my-keychain-writer-name)` to get the Keychain Writer address by its name. Alternatively, you can just specify the address obtained in the previous step.
    :::
 
+4. Finally, add the Writer account to your Keychain. Specify the Keychain ID from [Step 2.3](#2-register-a-keychain) and other details:
+   
+   ```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)